doc/releases/migration-guide-3.7.rst

```diff
@@ -221,6 +221,16 @@ Serial
 Timer
 =====
 
+Watchdog
+========
+
+* The ``nuvoton,npcx-watchdog`` driver has been changed to extend the max timeout period.
+  The time of one watchdog count bacomes various with the different pre-scalar setting.
```

Comment:
"bacomes various with the" 
Typo and weird grammar.

Reply:
done